ui/apps/platform/src/Containers/Dashboard/PatternFly/DashboardPage.css

```diff
@@ -0,0 +1,13 @@
+.widget-grid {
+  --min-widget-width: 510px;
```

Comment:
This should prevent headers from overflowing on widgets until the "# Active aging images widget" exceeds 10,000 images total in the title.